### PR TITLE
make mx_SenderProfile inline-block, reduces clickable area around usernames

### PR DESCRIPTION
**Pull request by benparsons**

### CODE REVIEW

**Comment by turt2live** [comment]
@benparsons is there an issue we can associate with this? I'm not sure what "stops accidental name inserting" means, sadly ._.

**Comment by benparsons** [comment]
Oops, yes it can be linked to this issue: https://github.com/vector-im/riot-web/issues/5400

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -45,14 +45,14 @@ limitations under the License.
...
-    display: block; /* anti-zalgo, with overflow hidden */
+    display: inline-block; /* anti-zalgo, with overflow hidden */
     overflow-y: hidden;
     cursor: pointer;
     padding-left: 65px; /* left gutter */
     padding-bottom: 0px;
     padding-top: 0px;
     margin: 0px;
-    line-height: 22px;
+    line-height: 17px;
```
Comment (jryans) [inline res/css/views/rooms/_EventTile.scss]: What's the reason for changing this value? (I guess I am not sure why either 22 or 17 is a good value here, but anyway it seems unrelated to the main issue you want to fix.)

Comment (benparsons) [reply res/css/views/rooms/_EventTile.scss]: Visually, Firefox and Chrome don't have quite the same line-height right now because it looks like FF is not using that field anyway in this case. Changing 22px to 17px makes them look visually the same *after* changing the display value.

Comment (jryans) [reply res/css/views/rooms/_EventTile.scss]: Okay, thanks for the detail. Seems reasonable to me then.
